fic_modules/helper_functions.py

```diff
@@ -102,26 +109,30 @@ def get_commit_details(commit):
 def extract_email(commit_email):
     """
     Helper function!
-    Takes as parameter a string that contains between "<" and ">" an email that needs to be
-    extracted.
-    The function uses find to search for the beginning of the email (that starts with "<") and adds
-    the lengths of the "<" so that returned email doesn't contain the "<" symbol and rfind to find
-    the ending character ">".
-    Both find and rfind return the index where the carachters "<" and ">" are placed so when you do
-    return commit_email with [start_char_index:ending_char_index] the string shrinks to what's
+    Takes as parameter a string that contains between "<" and ">" an email
+    that needs to be extracted.
+    The function uses find to search for the beginning of the email (that
+    starts with "<") and adds the lengths of the "<" so that returned email
+    doesn't contain the "<" symbol and rfind to find the ending character ">".
+    Both find and rfind return the index where the carachters "<" and ">" are
+    placed so when you do return commit_email with
+    [start_char_index:ending_char_index] the string shrinks to what's
     between the characters.
     :param commit_email: String that contains an email
     :return: String that contains only the email
     """
-    return commit_email[commit_email.find("<") + len("<"):commit_email.rfind(">")]
+    return commit_email[commit_email.find("<") + len("<"):commit_email.rfind(
+        ">")]
```

Comment:
This break here is pretty ugly.
Can we please move to a new line after the `+` sign?

Reply:
Fixed.